DesktopModules/Vanjaro/UXManager/Library/Base.cshtml

```diff
@@ -70,7 +67,7 @@
                     <div class='blockstabview'>
                         <div class='elementtab uitab blockstab active'><a><em class='fas fa-cube'></em><br />[LS:Blocks]</a></div>
                         <div class='customtab uitab blockstab'><a><em class='fas fa-box'></em><br />[LS:Custom]</a></div>
-                        @if (Model.Setting.ShowUXManagerToolbar)
+                        @if (Model.Setting.InitTabGrapesjs)
```

Comment:
Why was this changed? ShowUXManangerToolbar is much more intuitive than InitTabGrapesjs